src/encoded/reports/batch_download.py

```diff
@@ -47,7 +47,7 @@ def _get_metadata_link(self):
         )
 
 
-class BatchDownload(MetadataReport, BatchDownloadMixin):
+class BatchDownload(BatchDownloadMixin, MetadataReport):
```

Comment:
is this cosmetic or does it actually matter?  Usually I expect the mixin after.

Reply:
I think in this case it's important that the method resolution hits the mixin first, and then hits metadata if nothing is found.